Training/Week2/src/exercise11.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
The solution is correct although the rest of the code is missing in order to run it.